### PR TITLE
Added regression scoring metrics + support for random loading of validation examples

### PR DESCRIPTION
**Pull request by rmovva**
1. Additional regression scoring metrics, "pearson_corr" and "mean_squared_error"
2. Instead of loading first N validation examples to score model, can load a (reproducible) random sample with max_to_load_is_random=True parameter in valid_data_loader_config. (Set random seed with seed parameter, default 1)

### CODE REVIEW

File: momma_dragonn/model_creators/flexible_keras.py [hunk 1]
```diff
@@ -93,8 +93,11 @@ def _get_uncompiled_model(self, seed):
         import numpy as np
         np.random.seed(seed)
         import keras
+        # keras Graph module is deprecated in keras 1
         from keras.models import Graph
         graph = Graph()
+        #from keras.models import Model
```
Comment (AvantiShri) [inline momma_dragonn/model_creators/flexible_keras.py]: Changes look fine - can you remove stray comments from the version you actually commit?

File: momma_dragonn/model_trainers/keras_model_trainer.py [hunk 2]
```diff
@@ -124,6 +124,9 @@ def on_epoch_end(self, epoch, logs=None):
             csv_logger=CSVLogger(self.csv_logger,append=True)
             extra_callbacks.append(csv_logger) 
         try:
+            #print(self.samples_per_epoch)
```
Comment (AvantiShri) [inline momma_dragonn/model_trainers/keras_model_trainer.py]: Here's the other place with stray comments

File: scripts_test/config/end_of_training_callbacks_config.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
     {"class":"WriteToDbCallback",
      "kwargs":{"db_path": "runs.db"}},
     {"class":"EmailCallback",
-     "kwargs":{"to_emails":["avanti@stanford.edu"],
+     "kwargs":{"to_emails":["therajiv@gmail.com"],
```
Comment (AvantiShri) [inline scripts_test/config/end_of_training_callbacks_config.yaml]: Might want to leave that as my email in the committed version unless you want to get random emails from people using the framework for the first time :-)

**Comment by rmovva** [comment]
Should be good now.
